DataFormats/Phase2HLTPFTaus/interface/PFTauPairFwd.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DataFormats_Phase2HLTPFTaus_PFTauPairFwd_h
```

Comment:
Alternatively, drop the `#include "DataFormats/Phase2HLTPFTaus/interface/PFTauPair.h"` statement and use an actual forward declaration for `PFTauPair`, _e.g._
```c++
namespace reco {
    class PFTauPair;
    using PFTauPairCollection = std::vector<PFTauPair>;
}
```
though I'm not sure if it will work, or if it is actually useful.